articles/tokens/access-token.md

```diff
@@ -6,15 +6,29 @@ toc: true
 
 ## Overview
 
-The Access Token, commonly referred to as `access_token` in code samples, is a credential that can be used by a client to access an API. The `access_token` should be used as a `Bearer` credential and transmitted in an HTTP `Authorization` header to the API. Auth0 uses access tokens to protect access to the Auth0 Management API.
+The Access Token, commonly referred to as `access_token` in code samples, is a credential that can be used by a client to access an API. The `access_token` can be any type of token (such as an opaque string, or a JWT) and is meant for the API. The purpose of the `access_token` is to inform the API that the bearer of this token has been authorized to access the API and perform specific actions (as specified by the `scope` that has been granted). The `access_token` should be used as a `Bearer` credential and transmitted in an HTTP `Authorization` header to the API. 
 
-Auth0 generates access tokens in [JSON Web Token (JWT)](/jwt) format, an industry standard. JWTs contain three parts: a header, a set of claims, and a signature:
-- The header contains metadata about the type of token and the cryptographic algorithms used to secure its contents.
-- The set of claims contains verifiable security statements such as the identity of the user and the permissions they are allowed.
-- The signature is used to validate that the token is trustworthy and has not been tampered with.
+## Access Tokens at Auth0
 
-::: note
-The [Auth0 Management API v1](/api/management/v1) (which has been deprecated) uses an opaque token format in which claims are referenced in a separate database, rather than directly in the token.
+Auth0 currently generates access tokens in two formats:
+
+* As opaque strings, when `${account.namespace}/userinfo` is the audience.
+* As a [JSON Web Token (JWT)](/jwt) when a custom API is specified as the audience.
+
+JWTs are an industry standard, and are the format access tokens are generated in by Auth0 when a custom API is the audience. When a custom API audience is specified along with an `openid` scope, an access token is generated that will be valid for both the `/userinfo` endpoint and for the custom API. 
```

Comment:
The first sentence of this paragraph (JWTs are an industry standard...) says the same thing as the previous bullet, I'd remove it